fl4health/clients/basic_client.py

```diff
@@ -44,7 +45,7 @@ def __init__(
             checkpointer (Optional[TorchCheckpointer], optional): Checkpointer to be used for client-side
                 checkpointing. Defaults to None.
         """
-        super().__init__(data_path, device)
+        super().__init__(data_path, device, seed=seed)
```

Comment:
It looks like the seed setting still resides in the NumpyFLClient. As noted in the PR description, since this is going to be merged with BasicClient, let's move the seed logic into this class?

Reply:
To incorporate his changes you can simple merge in his branch and set the target of this PR to his branch until it is merged.

Reply:
Yes in current version I moved it into the basic client.